guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -1,20 +1,19 @@
 [id="Creating_an_Activation_Key_{context}"]
 = Creating an activation key
 
-Subscription Manager attaches subscriptions differently depending on the following factors:
+Activation key attaches subscriptions automatically.
 
-* Are there any subscriptions associated with the activation key?
-* For {RHEL} 8 hosts: Is there system purpose set on the activation key?
+. Activation key providing a custom set of subscription for auto-attach.
+If there are subscriptions specified and auto-attach is enabled, hosts using the activation key select the best fitting subscription from the list specified in the activation key.
+Setting system purpose on the activation key does not affect this scenario.
+. Activation key with the exact set of subscriptions.
+If there are subscriptions specified and auto-attach is disabled, hosts using the activation key are associated with all subscriptions specified in the activation key.
+Setting system purpose on the activation key does not affect this scenario.
 
-Note that {Project} automatically attaches subscriptions only for the products installed on a host.
-For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.
+.Custom Products
+If a {customproduct}, typically containing content not provided by Red Hat, is assigned to an activation key, this product is always enabled for the registered content host regardless of the auto-attach setting.
```

Comment:
Remove all of this (lines 4-16, from "Activation key attaches subscriptions automatically" to and including "If a {customproduct}..."